app/controllers/users_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=User%20logs%20in%20with%20a%20combination%20of%20email%20and%20password.) the user suppose to log in first to see the main page and that's not happening, Please fix that.
[Hint] You can add `before_action :authenticate_user!` at the beginning of the `UsersController` class.